snap7/common.py

```diff
@@ -87,3 +90,9 @@ def error_text(error, context: str = "client") -> bytes:
     elif context == "partner":
         library.Par_ErrorText(error, text, len_)
     return text.value
+
+
+def find_locally(fname):
+    if os.path.isfile("./" + fname + ".dll"):
```

Comment:
i think you can make this more robust by using `pathlib.Path(__file__) / 'snap7.dll'` or something similar (maybe syntax incorrect)

Reply:
Ok